template/lib.typ

```diff
@@ -227,27 +227,151 @@ text)
   }
 
 
+#let pretty-header(numeration: "1", main: false) = {
+  if(main){
+      context {
+    // Header for the main content
+    let titles = query(selector(heading.where(level: 1).or(heading.where(level:2))))
+
+    let title_left
+    let title_right
+    let chapter 
```

Comment:
Actual question, does `let x` in Typst mean that `x` gets assigned to `none`? Hadn't seen this before